tests/functions_tests/test_poolings.py

```diff
@@ -78,3 +80,45 @@ def __call__(self, x):
             return self.ops(*([x] + self.args), cover_all=self.cover_all)
         else:
             return self.ops(*([x] + self.args))
+
+
+class TestROIPooling2D(unittest.TestCase):
+
+    def setUp(self):
+        # these parameters are referenced from chainer test
+        in_shape = (3, 3, 12, 8)
+        self.x = input_generator.positive_increasing(*in_shape)
+        # in chainer test, x is shuffled and normalize-like conversion,
+        # in this test, those operations is skipped.
+        # if x includes negative value, not match with onnxruntime output.
```

Comment:
IIUC, changing `positive_increasing` to `input_generator.increasing` exposes the error? If it's the case, how about adding something like "You can reproduce this issue by changing `positive_increasing` to `increase`"?

Reply:
exactly, I added